index.html

```diff
@@ -489,48 +489,49 @@ <h3>Services</h3>
+Coordinators and returns well-formed, signed Verifiable Credentials. This
+service MUST have access to private keys (or key services which utilize private
+keys) in order to create the proofs for those VCs. The API between the Issuer
+service and its associated key service is believed to be out of scope for the VC
+API, but may be addressed by WebKMS or similar specifications.
         </p>
         <p>
-The Verifier service takes requests to verify Verifiable Credentials and
-Verifiable Presentations and returns the result of checking their proofs
-and status (if present). The service only checks the authenticity and
-timeliness of the VC; leaving the Verifier Coordinator to finish Applying any business rules needed.
+The <dfn>verifier service</dfn> takes requests to verify Verifiable Credentials
+and Verifiable Presentations and returns the result of checking their proofs and
+status (if present). The service only checks the authenticity and timeliness of
+the VC; leaving the Verifier Coordinator to finish Applying any business rules
+needed.
         </p>
         <p>
-  The Holder service takes requests to create Verifiable Presentations
-  from an optional set of VCs and returns well-formed, signed Verifiable
-  Presentations containing those VCs. These VPs are used with Issuers to
-  demonstrate control over DIDs prior to issuance and with Verifiers to
-  present specific VCs.
+The <dfn>holder service</dfn> takes requests to create Verifiable Presentations
+from an optional set of VCs and returns well-formed, signed Verifiable
+Presentations containing those VCs. These VPs are used with Issuers to
+demonstrate control over DIDs prior to issuance and with Verifiers to present
+specific VCs.
         </p>
       </section>
       <section>
         <h3>Status Service</h3>
         <p>
-        The Status Service provides a privacy-preserving means for publishing
-        and checking the status of any Verifiable Credentials issued by the
-        Issuer. Implementers of verifier services are encouraged to understand the
-        privacy implications of checking status by referring to the respective
-        status specification used by the verifiable credential.
+The <dfn>status service</dfn> provides a privacy-preserving means for publishing
+and checking the status of any Verifiable Credentials issued by the Issuer.
+Implementers of verifier services are encouraged to understand the privacy
+implications of checking status by referring to the respective status
+specification used by the verifiable credential.
         </p>
         <p>
-          For specific mechanisms by which to manage Verifiable Credential statuses,
-          it's recommended to refer to external well known specifications, such as the
-          [[VC-BITSTRING-STATUS-LIST]].
+For specific mechanisms by which to manage Verifiable Credential statuses, it's
+recommended to refer to external well known specifications, such as the
```

Comment:
```suggestion
recommended to refer to external well-known specifications, such as the
```

Reply:
The word order I suggest makes it (somewhat) less likely that `well-known specifications` will be read as a unit, thus not referencing only specifications that relate to the `.well-known` space.
```suggestion
recommended to refer to well-known external specifications, such as the
```